admin/acceptance.py

```diff
@@ -950,6 +951,98 @@ def eliot_output(message):
     sys.stdout.write(format % message)
     sys.stdout.flush()
 
+def capture_upstart(reactor, host, output_file):
+    """
+    SSH into given machine and capture relevant logs, writing them to
+    output file.
+
+    :param reactor: The reactor.
+    :param bytes host: Machine to SSH into.
+    :param file output_file: File to write to.
+    """
+    formatter = TailFormatter(output_file, host)
```

Comment:
You're not passing in `service` here, so it'll always be "unknown". You should be able to get service by tailing each file separately and having a TailFormatter for each.